accepted/0000-dependency-library.md

```diff
@@ -0,0 +1,38 @@
+# Dependency Library to Provide Dependency Info
+
+## Summary
+
+The dependency metadata that is provided by the [Dep Server](https://github.com/paketo-buildpacks/dep-server) is backed partially by a Go library which provides information about upstream dependencies (available versions, release dates, source code URLs and SHAs, etc.). This RFC proposes creating a new repo that will expose this underlying resource.
+
+## Motivation
+
+By exposing this library, buildpack authors who use the Dep Server will have access to and insight into more of the underlying resources backing the server.
```

Comment:
Can you elaborate a bit more about how/when paketo buildpack authors are currently using the dep server? Selfishly, given the Java buildpacks don't use the dep server, I am not as familiar with it. Capturing some of the background/context in the RFC might be good for posterity as well.

Reply:
I can't speak to who is using the server yet, but they would use it to find dependencies that have compiled and uploaded to a public S3 bucket so that they can include them in their buildpack.toml. I think [this RFC](https://github.com/paketo-buildpacks/rfcs/blob/master/accepted/0011-dep-server.md#detailed-explanation) might help explain how they might use it.

Reply:
We are not yet using the Dependency Server for the other language family buildpacks yet. This is [definitely in](https://github.com/paketo-buildpacks/feedback/discussions/2#discussioncomment-263526) the Roadmap for 2021 though. Having a client library for this would help a bit to enable developing this type of automation.